webpack/private/webpack_bundle.bzl

```diff
@@ -421,3 +423,36 @@ def webpack_bundle(
         allow_execroot_entry_point_with_no_copy_data_to_bin = allow_execroot_entry_point_with_no_copy_data_to_bin,
         **kwargs
     )
+
+def webpack_binary(name, node_modules):
+    """Create a webpack binary target from linked node_modules in the user's workspace.
+
+    Pass this into the `webpack` attribute of webpack_bundle to use your own linked
+    version of webpack rather than rules_webpack's version, which can help to avoid
+    certain errors caused by having two copies of webpack. The following three packages
+    must be linked into the node_modules virtual store target:
+
+        webpack, webpack-cli, webpack-dev-server
+
+    Args:
+        name: Unique name for the binary target
+        node_modules: Label pointing to the linked node_modules target where
+            webpack is linked, e.g. `//:node_modules`.
+    """
+
+    directory_path(
```

Comment:
hmmm. incase `node_modules` is a label ... not likely but it could happen. it would be more robust to do the following

```
node_modules = to_label(node_modules)
```

```
directory = paths.join(node_modules.package, node_modules.name, "webpack/dir")
```

etc...

and also sanity check that the label is in the same workspace as the `webpack_binary` macro.

Reply:
to_label is in our bazel-lib, and paths is in skylib

Reply:
or maybe this is find the way it is since the to string of a label joined with `/anything` should still be valid